lib/ansible/modules/network/nxos/nxos_interface.py

```diff
@@ -548,8 +547,21 @@ def smart_existing(module, intf_type, normalized_interface):
 
 def execute_show_command(command, module):
     if module.params['transport'] == 'cli':
```

Comment:
You don't have to split on `transport`, as both ways work for both transports, though we're trying to standardize on the latter (`output = 'whatever'`) as it is a bit clearer what is intended, and allows one to specify text output directly